### PR TITLE
New model integration

### PR DESCRIPTION
**Pull request by codymccoy**

### CODE REVIEW

**Review by RyanM-RMA — CHANGES_REQUESTED** [review]
Few things to resolve, let me know when it's ready

File: ViewModel/ImpactAreaScenario/SpecificIAS.cs [hunk 1]
```diff
@@ -275,5 +305,9 @@ private ThresholdType ConvertStringToMetricEnum(string metric)
             }
         }
 
+        public void ReportMessage(object sender, MessageEventArgs e)
```
[the remark this answers is not in the document]
Comment (codymccoy) [reply ViewModel/ImpactAreaScenario/SpecificIAS.cs:308]: i removed it

File: ViewModel/TableWithPlot/Data/UniformDataProvider.cs [hunk 2]
```diff
@@ -10,17 +10,28 @@ internal class UniformDataProvider: BaseDataProvider
...
+            IsStrictMonotonic = false;
             Name = "Uniform";
-            Data.Add(new UniformRow(0.0d, new Uniform(0, 0)));
-            Data.Add(new UniformRow(2.0d, new Uniform(0, 2)));
+            Data.Add(new UniformRow(0.0d, new Uniform(0, 0), IsStrictMonotonic));
+            Data.Add(new UniformRow(2.0d, new Uniform(0, 2), IsStrictMonotonic));
             LinkList();
         }
-        public UniformDataProvider(UncertainPairedData upd)
+        public UniformDataProvider(bool isStrictMonotonic)
```
[the remark this answers is not in the document]
Comment (codymccoy) [reply ViewModel/TableWithPlot/Data/UniformDataProvider.cs:19]: Yeah, i wasn't really sure how to handle this one. This is part of the table with plot that HEC wrote and that i copied into FDA. The empty ctor is required for some kind of magic. I defaulted the property and removed the assignment from the empty ctor.

File: ViewModel/ImpactAreaScenario/Editor/SpecificIASEditorVM.cs [hunk 3]
```diff
@@ -37,6 +37,12 @@ public class SpecificIASEditorVM : BaseViewModel
         private double _EAD;
         private PairedData _DamageFrequencyCurve = null;
 
+        public string FrequencyRelationshipLabel { get; } = StringConstants.FREQUENCY_RELATIONSHIP_LABEL;
```
[the remark this answers is not in the document]
Comment (codymccoy) [reply ViewModel/ImpactAreaScenario/Editor/SpecificIASEditorVM.cs:40]: you will need to show me this one. Can i keep the strings in the VM or do i need to move them into the view?

**Review by RyanM-RMA — APPROVED** [review]

**Pull request merged**